src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
What's complicated is that decreasing `r` reduces the likelihood of a fail-high (which triggers a full-depth search).  It's true that capture races may occur but in general, if the opponent has to capture on his next move it is improbable (yet possible) that our next move is a capture.

Honestly what's most complicated about this isn't so much the `captureOrPromotion` case (since promotions are useful) but that which follows regarding `ttCapture` and moves that escape a capture (both of which sound undesirable for giveaway chess - in cases where you're capturing by transposition and/or evading capture, at least in my experience you'd want to double-check those variants with extra effort, not reduced effort - but maybe for engines it's different).